releases.yml

```
@@ -2,28 +2,28 @@ releases:
   4.8.57:
     assembly:
       basis:
-        brew_event: 49280102
+        brew_event: 49317966
         reference_releases:
-          ppc64le: 4.8.0-0.nightly-ppc64le-2023-01-17-103232
-          s390x: 4.8.0-0.nightly-s390x-2023-01-17-103230
-          x86_64: 4.8.0-0.nightly-2023-01-17-103230
+          ppc64le: 4.8.0-0.nightly-ppc64le-2023-01-20-013809
+          s390x: 4.8.0-0.nightly-s390x-2023-01-20-013808
+          x86_64: 4.8.0-0.nightly-2023-01-20-013809
       group:
         advisories:
-          extras: 108394
-          image: 108393
-          metadata: 108395
-          rpm: 108392
-        release_jira: ART-5831
+          extras: -1
+          image: -1
+          metadata: -1
+          rpm: -1
+        release_jira: ART-0
```

Good except this bit
```suggestion
          extras: 108394
          image: 108393
          metadata: 108395
          rpm: 108392
        release_jira: ART-5831
```